<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>sstvDecoder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureMode">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="monoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>mono</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 channel</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="stageLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ifdisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="snrdisplay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
